content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -31,15 +26,10 @@ Datadog instrumentation to prevent duplicate spans from appearing in the trace.
 
 ## Configuration
 
-Datadog SDKs support configuration through [OpenTelemetry environment variables][16].
+Datadog SDKs also support configuring [OpenTelemetry environment variables][16].
```

Comment:
I'm not sure about this new phrasing. The SDKs don't "support configuring OpenTelemetry environment variables." Users configure the tracer by setting environment variables.

Maybe reword this to something like:
> You can also configure Datadog SDKs by setting the same environment variables supported by OpenTelemetry.

or

> The Datadog SDKs support many of the same environment variables supported by OpenTelemetry.

Reply:
Yea. Making this change